doc/data-loss.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting data loss with PPK2
```

Comment:
Should this perhaps be a more general FAQ document with a data loss section? Might be useful for other stuff also, for example how to potentially handle save files that cannot be opened.

Reply:
Good idea.